test/action_controller/json_api/transform_test.rb

```diff
@@ -77,7 +77,7 @@ def test_render_resource_with_transform
           expected = {
             'Data' => {
               'Id' => '1337',
-              'Type' => 'Posts',
+              'Type' => 'posts',
```

Comment:
If it's preferential for me to change the config for this test, and keep the payload the same, I can do that